static/app/views/profiling/content.tsx

```diff
@@ -127,7 +129,19 @@ function ProfilingContent({location, router}: ProfilingContentProps) {
           <StyledPageContent>
             <Layout.Header>
               <StyledLayoutHeaderContent>
-                <StyledHeading>{t('Profiling')}</StyledHeading>
+                <StyledHeading>
+                  {t('Profiling')}
+                  <PageHeadingQuestionTooltip
+                    title={tct(
+                      'A view of how your application performs in a variety of environments, based off of the performance profiles collected from real user devices in production. [link: Read the docs].',
+                      {
+                        link: (
+                          <ExternalLink href="https://dev.getsentry.net:7999/organizations/sentry/profiling/" />
```

Comment:
Should this link to https://docs.sentry.io/product/profiling/

Reply:
Oof, good catch! Clearly had the wrong thing on my clipboard there...